dev/io.openliberty.io.opentelemetry/bnd.bnd

```diff
@@ -41,7 +41,7 @@ Import-Package: \
 
 Export-Package: \
   io.opentelemetry.extension.trace.propagation;version=${openTelemetryVersion};thread-context=true,\
-  io.opentelemetry.*;version=${openTelemetryVersion}
+  io.opentelemetry.*;version=${openTelemetryVersion};thread-context=true
```

Comment:
Why is this needed? I don't think it should be necessary.

Reply:
This is needed or I would get this error
```
[ERROR   ] CWMOT5002E: An internal error occurred in MicroProfile Telemetry. The error is: io.opentelemetry.sdk.autoconfigure.spi.ConfigurationException: OTLP gRPC Trace Exporter enabled but opentelemetry-exporter-otlp not found on classpath. Make sure to add it as a dependency to enable this feature.class not found: io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter.
```

Reply:
I have created another PR https://github.com/OpenLiberty/open-liberty/pull/26278.  We can continue to discuss there.